src/plugins/controls/public/control_group/editor/control_editor.tsx

```diff
@@ -211,6 +214,23 @@ export const ControlEditor = ({
               {ControlGroupStrings.manageControl.getCancelTitle()}
             </EuiButtonEmpty>
           </EuiFlexItem>
+          <EuiFlexItem grow />
+          <EuiFlexItem grow={false}>
```

Comment:
Talked with Andrea about this [after I logged the following bug](https://github.com/elastic/kibana/issues/131874) and she mentioned that you guys agreed to keep the delete button in its original place - I assume this was left in by accident? Will close the bug - didn't realize it was caused by this code, haha! Oops 😆 But yeah, we should move the "Delete" button back to its original place.